services/webhook/tests/test_rerun.py

```
@@ -908,3 +908,38 @@ def json(self):
+        return Resp()
+
+    monkeypatch.setattr(rerun, "with_install_token_retry", lambda iid, fn: fn("tok"))
+    monkeypatch.setattr(rerun.httpx, "get", fake_get)
+    reason = rerun._elder_check_already_terminal_or_pending(
+        install_id=1, owner="o", repo_name="r", head_sha="h" * 40,
+    )
+    assert reason == "already_completed_success"
+    # Guard the cutover contract server-side too: a check_name param would
+    # make GitHub filter OUT the legacy alias run and this test would pass
+    # for the wrong reason.
+    assert seen_params and "check_name" not in seen_params[0]
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)